docs/update-locking.md

```diff
@@ -16,9 +16,33 @@ assumption that updates are locked (see [issue #20](https://github.com/resin-io/
 
 In supervisors >= v4.0.0 and any OS that is not Resin OS 1.X, the old lock location is completely ignored.
 
-### Taking the lock
+### Creating the lockfile
 
-Using the above-mentioned library, the lock can be acquired like in this CoffeeScript example:
+There are many different tools and libraries to provide proper lockfile functionality. Here are a few examples, and you can search online for others in other programming languages not mentioned here explicitly.
+
+__Note:__ Just creating the lockfile (for example by `touch /tmp/resin/resin-updates.lock`) is not adequate to prevent updates. That file way won't have the exclusive access flag set, and thus does not provide reliable locking.
+
+#### Shell
+
+One simple way to take the lock is using [lockfile](https://linux.die.net/man/1/lockfile) (available for example in Debian from the `procmail` package) or [flock](https://linux.die.net/man/1/flock) (available for example in Debian from the `linux-utils` package). Some example use are:
```

Comment:
NIT
I would restructure this `Shell` section so that the lockfile example is immediately below the lockfile description and the same with the flock example

Reply:
Done